packages/web/src/Admin_dashboard_routes.js

```diff
@@ -0,0 +1,83 @@
+        <Route path="members/add" element={<BackBtnAdd Component={AddMember} />} />
+        <Route path="jobs" element={<BackBtnAdd Component={RenderJobs} />} />
+        <Route path="jobs/actions" element={<BackBtnAdd Component={JobActions} />} />
+        <Route path="mentors" element={<BackBtnAdd Component={RenderMentors} />} />
+        <Route path="mentors/add" element={<BackBtnAdd Component={AddMentors} />} />
+        <Route path="mentors/actions" element={<BackBtnAdd Component={MentorsActions} />} />
+        <Route path="jury" element={<BackBtnAdd Component={RenderJury} />} />
+        <Route path="jury/add" element={<AddJury />} />
+        <Route path="jury/actions" element={<BackBtnAdd Component={JuryActions} />} />
+        <Route path="sponsors" element={<BackBtnAdd Component={RenderSponsors} />} />
+        <Route path="sponsors/add" element={<BackBtnAdd Component={AddSponsors} />} />
+        <Route path="sponsors/actions" element={<BackBtnAdd Component={SponsorsActions} />} />
+        <Route path="partners" element={<BackBtnAdd Component={RenderPartners} />} />
+        <Route path="partners/add" element={<BackBtnAdd Component={AddPartners} />} />
+        <Route path="partners/actions" element={<BackBtnAdd Component={PartnersActions} />} />
+        <Route path="hackathon/teams" element={<BackBtnAdd Component={RenderTeams} />} />
+        <Route path="hackathon/teams/members" element={<BackBtnAdd Component={RenderTeamMembers} />} />
+        <Route path="hackathon/teams/members/actions" element={<BackBtnAdd Component={TeamMemberActions} />} />
+        <Route path="hackathon/teams/members/add" element={<BackBtnAdd Component={AddTeamMember} />} />
+        <Route path="hackathon/teams/add" element={<BackBtnAdd Component={AddNewTeam} />} />
+        <Route path="hackathon/noteamparticipants/add" element={<BackBtnAdd Component={AddNoTeamParticipant} />} />
+        <Route path="hackathon/teams/actions" element={<BackBtnAdd Component={TeamActions} />} />
+        <Route path="hackathon/noteamparticipants" element={<BackBtnAdd Component={RenderNoTeamParticipants} />} />
+        <Route
+            path="hackathon/noteamparticipants/actions"
+            element={<BackBtnAdd Component={NoTeamParticipantsActions} />}
+        />
+        <Route path="jobs" element={<BackBtnAdd Component={RenderJobs} />} />
+        <Route path="jobs/actions" element={<BackBtnAdd Component={JobActions} />} />
+        <Route path="jobs/add" element={<BackBtnAdd Component={AddJobs} />} />
+        <Route path="s3" element={<BackBtnAdd Component={S3Panel} />} />
+        <Route path="s3/objects" element={<BackBtnAdd Component={RenderStorageObjects} />} />
+        <Route path="shortener" element={<BackBtnAdd Component={UrlsTable} />} />
+        <Route path="fswitches" element={<BackBtnAdd Component={RenderSwitches} />} />
+    </Routes>
+);
+const BackBtnAdd = ({ Component }) => {
```

Comment:
Only component names are capitalized. Rename the prop to `component` or `element`.

Reply:
But if is not capitalized it does not accept is as a React component.
![image](https://github.com/AUBGTheHUB/monolith/assets/99684754/8a7bb23b-1d7b-4afb-b95a-582e376ce9d4)
![image](https://github.com/AUBGTheHUB/monolith/assets/99684754/eb038e92-2431-42d7-8003-322f6fb6c097)
Do you know of a way around it? I tried searching but did not find anything

Reply:
```javascript
const AdminContainerWithButton = ({ component }) => {
    return (
        <>
            <BackBtn />
            {component}
        </>
    );
};
```

Reply:
Remind me next time to tell you a little bit more about what children are and how to pass them.